bokeh/resources.py

```diff
@@ -274,6 +274,25 @@ def _file_paths(self, kind):
         paths = [ join(bokehjs_dir, kind, file) for file in files ]
         return paths
 
+    def _collect_external_resources(self, resource_attr):
+        """ Collect external resources set on resource_attr attribute (expected
+        to be "__external_js_resources__" or "__external_css_resources__") of models.
+        """
+
+        external_resources = []
+
+        for cls in Model.model_class_reverse_map.values():
+            external = getattr(cls, resource_attr, None)
+
+            if isinstance(external, str):
```

Comment:
I imagined supporting both single and multiple external resources like:

```python
    class CustomModel(Model):
        __external_js_resources__ = "external_js_1"
        __external_css_resources__ = ["external_css_1", "external_css_2"]
```

Alternative we could require only lists, but this seems easy enough to support and one fewer gotcha for users.